Issue 5543: for theme

elzadj opened the pull request:
# Description

Fixes some points from #5543 
You need to edit your SC and activate it, or reset them all. Better edit some SC colour, reset it, and save the SC.

Fixes #5543

# How Has This Been Tested?

Test native pagination and prev / next for posts. 
Test inputs in comment form (not logged in).
Ignore failing tests for now.

# Test checklist

<!--- Please remove the unnecessary checkbox --->

**_ Front/Back Testing _**

-   [ ] Test the component in normal state in sidebar
-   [ ] Test the component in hover state in sidebar (if hover exists)
-   [ ] Check that the settings work on frontend
-   [ ] Check that backend works and saves the settings after the editor reload
-   [ ] Same 1-4 points for the toolbar
-   [ ] Same 1-4 points on responsive
-   [ ] Test the block inside the grid (Container + Row + Column)
-   [ ] Test the block as a standalone block
-   [ ] Duplicate the block, test that the settings of the first do not affect the second
-   [ ] Test with 2 blocks of the same type

**_ Pre-Code Testing _**

-   [ ] Test the component in normal state in sidebar
-   [ ] Test the component in hover state in sidebar (if hover exists)
-   [ ] Check that the settings work on frontend
-   [ ] Check that backend works and saves the settings after the editor reload
-   [ ] Same 1-4 points for the toolbar
-   [ ] Same 1-4 points on responsive
-   [ ] Test the block inside the grid (Container + Row + Column)
-   [ ] Test the block as a standalone block
-   [ ] Duplicate the block, test that the settings of the first do not affect the second
-   [ ] Test with 2 blocks of the same type
-   [ ] Check no commented code and no unnecessary imports
-   [ ] Standards of the project have been followed
-   [ ] No errors/warnings on console

# Checklist

-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] My changes generate no new warnings/errors
-   [ ] I have added/updated tests that prove my fix is effective or that my feature works
-   [ ] New and existing unit tests pass locally with my changes


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit


- **Style Updates**
	- Enhanced styling for form inputs, navigation links, and pagination elements.
	- Introduced new colors and hover effects for buttons.
	- Added new styles for form inputs and button colors.
	- Updated CSS selectors for form inputs, navigation links, and pagination numbers.
	- Adjusted button hover effects and form input backgrounds.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

elzadj committed:
Add coments input, post and page nav to SC

elzadj committed:
remove logs

elzadj committed:
fix tests

Olekrut reviewed (changes requested):
https://jam.dev/c/ee442c98-02e2-4824-bca2-a6778dba46f6
[native blocks.txt](https://github.com/maxi-blocks/maxi-blocks/files/14959982/native.blocks.txt)
Not sure that we need it, but 
1) Next previous post blocks have Font-family:inter, not roboto
2) input fields have BG yet
3) guess, need to add max-width:100%; for textarea
4) textarea for comment form has bg as well, but not inside native block - Comments

elzadj committed:
Fix styles for forms

elzadj commented:
> https://jam.dev/c/ee442c98-02e2-4824-bca2-a6778dba46f6 [native blocks.txt](https://github.com/maxi-blocks/maxi-blocks/files/14959982/native.blocks.txt) Not sure that we need it, but
> 
> 1. Next previous post blocks have Font-family:inter, not roboto
> 2. input fields have BG yet
> 3. guess, need to add max-width:100%; for textarea
> 4. textarea for comment form has bg as well, but not inside native block - Comments

1. 
![Screenshot from 2024-04-12 17-58-50](https://github.com/maxi-blocks/maxi-blocks/assets/2401618/afb07e57-0446-4d1d-b934-aa2d50d45e82)
2. Fixed
3. Added
4. I guess fixed too, needs testing

You need to edit the SC again.

elzadj committed:
fix tests

Olekrut reviewed (changes requested):
Idk, lets test it on staging, because nothing is fixed for me, and styles on frontend are not working all the time
https://jam.dev/c/447c748a-9629-443d-9b6c-56effc35c339
